uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-notification-badge.component.js

```diff
@@ -37,7 +37,7 @@ define([
     function fetchCount () {
       vm.loading.count = true;
 
-      return LeaveRequest.all(vm.filters)
+      return LeaveRequest.all(vm.filters, null, null, null, false)
```

Comment:
Yes its for caching only. And I agree with you that current way of sending params has this disadvantage and this was discussed before too. But not sure when we will be able to refactor all the models/api's. cc @AkA84